src/main/java/com/mcmoddev/mmdbot/core/BotConfig.java

```diff
@@ -341,44 +372,59 @@ public long getCommunityChannelCategory() {
         return SafeIdUtil.safeConvert(getAliased("community_channels.category", getAliases()));
     }
 
+    /**
+     *
+     * @return
+     */
     @SuppressWarnings("unchecked")
     public EnumSet<Permission> getCommunityChannelOwnerPermissions() {
-        if (!config.contains("community_channels.owner_permissions")) {
+        if (!config.contains(COMMUNITY_CHANNEL_OWNER_PERMISSIONS)) {
             return EnumSet.noneOf(Permission.class);
         }
-        Object obj = config.get("community_channels.owner_permissions");
+        final Object obj = config.get(COMMUNITY_CHANNEL_OWNER_PERMISSIONS);
         if (obj instanceof Number) {
             return Permission.getPermissions(((Number) obj).longValue());
         } else if (obj instanceof List) {
-            List<String> permList = ((List<String>) obj);
-            EnumSet<Permission> permissions = EnumSet.noneOf(Permission.class);
+        	final List<String> permList = ((List<String>) obj);
+        	final EnumSet<Permission> permissions = EnumSet.noneOf(Permission.class);
             outer:
-            for (String perm : permList) {
-                for (Permission permission : Permission.values()) {
+            for (final String perm : permList) {
+                for (final Permission permission : Permission.values()) {
                     if (permission.getName().equals(perm) || permission.name().equals(perm)) {
                         permissions.add(permission);
                         continue outer;
                     }
                 }
-                MMDBot.LOGGER.warn("Unknown permission in \"community_channels.owner_permissions\": '{}'", perm);
+                MMDBot.LOGGER.warn("Unknown permission in \"" + COMMUNITY_CHANNEL_OWNER_PERMISSIONS + "\": '{}'", perm);
             }
             return permissions;
         }
-        MMDBot.LOGGER.warn("Unknown format of \"community_channels.owner_permissions\", resetting to blank list");
-        config.set("community_channels.owner_permissions", Collections.emptyList());
+        MMDBot.LOGGER.warn("Unknown format of \"" + COMMUNITY_CHANNEL_OWNER_PERMISSIONS + "\", resetting to blank list");
```

Comment:
I would suggest making this an argument instead and use parameter substitution (avoiding manual string concatentation).
```suggestion
        MMDBot.LOGGER.warn("Unknown format of \"{}\", resetting to blank list", COMMUNITY_CHANNEL_OWNER_PERMISSIONS);
```